js/ui/scheduler/ui.scheduler.appointments.strategy.base.js

```diff
@@ -559,14 +564,19 @@ var BaseRenderingStrategy = Class.inherit({
             appointmentCountPerCell = coordinates.count;
             ratio = (maxHeight - offsets.unlimited) / maxHeight;
         }
+
+        var topOffset = (1 - ratio) * maxHeight;
         if(overlappingMode === "auto") {
-            ratio = (maxHeight - offsets.auto) / maxHeight;
+            ratio = 1;
+            maxHeight = maxHeight - this._getAppointmentDefaultOffset();
+            topOffset = this._getAppointmentDefaultOffset();
```

Comment:
Don't call `_getAppointmentDefaultOffset` twice. Get result only once, next use it.